### PR TITLE
Add CWE, GDPR, ASVS and MSTG compliance

### PR DESCRIPTION
**Pull request by subho007**
Pending: Writing Testcase for the Compliances

### CODE REVIEW

File: app/models/analysis.js [hunk 1]
```diff
@@ -3,6 +3,10 @@ import { isEmpty } from '@ember/utils';
 import Model, { attr, hasMany, belongsTo }  from '@ember-data/model';
 import ENUMS from 'irene/enums';
 import { t } from 'ember-intl';
+import Inflector from  'ember-inflector';
+
+const inflector = Inflector.inflector;
+inflector.irregular('asvs', 'asvses');
```
Comment (subho007) [inline app/models/analysis.js:9]: @guganabu @cosmosgenius need to know one thing, I have written the irregular inflector not in app wide but places where it is required. Is this a better approach or do you want me to shift it to `app.js` ?

File: app/models/asvs.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import Model, { attr }  from '@ember-data/model';
+import Inflector from  'ember-inflector';
+const inflector = Inflector.inflector;
+
+inflector.irregular('asvs', 'asvses');
```
Comment (subho007) [inline app/models/asvs.js:5]: here too @guganabu @cosmosgenius

File: package.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
     "lint:hbs:fix": "ember-template-lint . --fix",
     "lint:js": "eslint . --cache",
     "lint:js:fix": "eslint . --fix",
-    "start": "cross-env IRENE_API_HOST=http://localhost:8000 ember server",
+    "start": "cross-env IRENE_API_HOST=http://localhost:8000 IRENE_API_SOCKET_PATH=http://localhost:8008 ember server",
```
Comment (cosmosgenius) [inline package.json:21]: `IRENE_API_SOCKET_PATH` is not used anymore...its not required

File: app/components/gdpr-details/template.emblem [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.columns
```
Comment (cosmosgenius) [inline app/components/gdpr-details/template.emblem:1]: this is a small template...can you move it to hbs...only for the new templates

**Review by cosmosgenius — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (subho007) [reply package.json]: Done 👍

File: app/models/analysis.js (hunk 1 above)
Comment (guganabu) [reply app/models/analysis.js:9]: I think, such conversions are done at per model level is better than having it in global file.

**Review by guganabu — COMMENTED** [review]
added comments

File: app/components/mstg-details/component.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+import Component from '@ember/component';
```
[the remark this answers is not in the document]
Comment (subho007) [reply app/components/mstg-details/component.js:1]: @guganabu without component script, the template doesn't load

[the remark this answers is not in the document]
Comment (subho007) [reply app/components/mstg-details/component.js]: This is done @guganabu 👍

**Review by Praseetha-KR — APPROVED** [review]

**Review by cosmosgenius — APPROVED** [review]

**Pull request merged**